nibabel/cmdline/tests/test_utils.py

```diff
@@ -11,7 +11,7 @@
 import nibabel as nib
 import numpy as np
 from nibabel.cmdline.utils import *
-from nibabel.cmdline.diff import get_headers_diff, display_diff, main, get_data_diff
```

Comment:
One of us has managed to make this file executable, please undo:
![capture _2018-09-27-20-06-42](https://user-images.githubusercontent.com/39889/46180899-fcf82480-c290-11e8-919b-913e870d4837.png)
If you like a challenge - undo by rewriting that original commit. Workflow:
- fix, commit
- `git rebase -i BADCOMMIT^` where you reposition fixing commit after the one to fix, and give it `s` status to squash them into one
- `git push -f` since now you rewritten a commit